crates/indexer/migrations/2022-09-29-073529_add_coin_tables/up.sql

```diff
@@ -0,0 +1,72 @@
+-- Your SQL goes here
+-- coin infos. Only first transaction matters
+CREATE TABLE coin_infos (
```

Comment:
ugh, why not coin_info is plural

Reply:
hehe. Legacy diesel requirement shit. I don't think that diesel 2.0 still requires it but when you have things like "metadatas" this doesn't seem that bad.